lib/sles4sap/sap_host_agent.pm

```diff
@@ -29,55 +28,6 @@ B<root> or B<sidadm>.
 
 =cut
 
-=head2 saphostctrl_list_databases
```

Comment:
The whole function is bad and returns passing value only by coincidence. 
`saphostctrl_list_instances` is the one that should be used.